examples/oft/hardhat.config.ts

```diff
@@ -67,6 +73,24 @@ const config: HardhatUserConfig = {
             url: process.env.RPC_URL_AMOY || 'https://polygon-amoy-bor-rpc.publicnode.com',
             accounts,
         },
+        'base-mainnet': {
+            eid: EndpointId.BASE_V2_MAINNET,
+            url: process.env.RPC_URL_BASE_MAINNET || 'https://base.drpc.org',
+            accounts,
+            safeConfig: {
+                safeAddress: '0x28937ca4873f7289Ebea0708c4E42b24835eCfF0',
+                safeUrl: 'https://safe-transaction-base.safe.global/',
+            },
+        },
+        'ethereum-mainnet': {
+            eid: EndpointId.ETHEREUM_V2_MAINNET,
+            url: process.env.RPC_URL_ETHEREUM_MAINNET || 'https://rpc.ankr.com/eth',
+            accounts,
+            safeConfig: {
+                safeAddress: '0xCDa8e3ADD00c95E5035617F970096118Ca2F4C92',
+                safeUrl: 'https://safe-transaction-mainnet.safe.global/',
+            },
+        },
```

Comment:
I can remove the safe configs here before merging -- just added here for quick testing. I obtained these safe addresses from the `ZROClaim` repo [here](https://github.com/LayerZero-Labs/ZROClaim/blob/d9acc7a2f54497670c71b11e47e098a688930d2c/hardhat.config.ts#L101-L122)